release_tools/README.adoc

```diff
@@ -0,0 +1,51 @@
+= How to perform a semi-automatic release =
+
+== Prerequisities ==
+
+* The `.env` file.
+  Create `.env` file in this directory (i.e. where the release scripts are).
+  You are supposed to define environment variables that contain sensitive information, s.a. GitHub access tokens.
+  Exact contents of the file are described below.
+
+* Github automation
+ * System with `python3.6` or greater, installed [PyGithub](https://pypi.python.org/pypi/PyGithub/1.35) module.
+ * A Github token - get one on https://github.com/settings/tokens and put it in the `.env` file, so it contains the line `GITHUB_TOKEN='<your token here>'`"
+
+== Process ==
+
+1. Clone a clean `openscap` repository:
+
+   git clone https://github.com/OpenSCAP/openscap.git
+
+2. Check out that the information within `versions.sh` is accurate.
++
+Openscap versions are supposed to have correct values from the previous build, but the `latest_fedora` and `latest_rhel` may need to be updated.
+
+3. Run `up_to_compliance_check.sh`.
++
+In the ideal case, the script ends after the ABI compatiblity check finishes.
+Review the check results and act accordingly to them.
+
+4. Run `handle_ltversions.sh`.
++
+According to the result from the compliance check (and the decision what to do about possible API/ABI changes), you run the script with an argument `bugfix`, `backwards_compatible` or `breaking_change`.
```

Comment:
It would be nice to explain that bugfix means no symbols added no symbols changed, no symbols removed, backwards mean some symbols were added and no symbols changed, no symbols removed and breaking change means that symobls were changed and removed.
It is kind of there but I think this is not sufficent for people doing release in future who don't know the background

Reply:
This has been addressed, I hope that I got it right.